aztec/src/test/kotlin/org/wordpress/aztec/AztecParserTest.kt

```diff
@@ -1024,7 +1026,8 @@ class AztecParserTest : AndroidTestCase() {
     @Throws(Exception::class)
     fun parseHtmlToSpanToHtmlLineBreakBetweenHeadings_isEqual() {
         val input = HTML_HEADING_ONE + "<br>" + HTML_HEADING_ONE
-        val span = SpannableString(mParser.fromHtml(input, null, null, context))
+        val span = SpannableStringBuilder(mParser.fromHtml(input, null, null, context))
+        mParser.syncVisualNewlinesOfBlockElements(span)
```

Comment:
Are this change (along with line 1150) still needed? The tests seem to pass without it anyway. Thanks!